internal/server/data/migrations.go

```diff
@@ -94,7 +94,7 @@ func migrate(db *gorm.DB) error {
 					key.IssuedFor = strings.TrimPrefix(key.IssuedFor, "u:")
 					key.IssuedFor = strings.TrimPrefix(key.IssuedFor, "m:")
 
-					iss, err := uid.ParseString(key.IssuedFor)
+					iss, err := uid.Parse([]byte(key.IssuedFor))
```

Comment:
What's the advantage of getting rid of `ParseString`?  Is converting to the byte slice more standard?

Reply:
My main motivation for removing this is that I always try to avoid functions that do almost the same thing. It becomes hard to use a package if there are a bunch of similar methods with only vary slight differences in args. If we expect `ParseString` to be called in a lot of places, I might be worth it to add.

I don't expect us to have to parse IDs in many places. Most of the places that I changed were in tests. Most of the time I'd expect JSON encode/decode to call parse for us, and it's pretty easy to cast a string to `[]byte` when we do need to call it.

I'm not against restoring `ParseString` if we think it'll be used more often.